can/io/canutils.py

```diff
@@ -161,7 +161,7 @@ def on_message_received(self, msg):
 
         channel = msg.channel if msg.channel is not None else self.channel
 
-        framestr = "(%f) %s" % (timestamp, channel)
+        framestr = "(%f) can%s" % (timestamp, channel)
```

Comment:
msg.channel might can be a string like `"can0"`. You'd get `"cancan0"` here.

Reply:
Good point!
Unfortunately I'm not really familiar with Python, but would it make sense to add some `isdigit()` vodoo to catch the case that we are only dealing with numbers and add `"can"` only in these cases?

Reply:
you could use
```python3
if isinstance(channel, int):
    ...
```

Reply:
Hi @zariiii9003 ,
I looked at the docs and also at some code - but I don't have Python knowledge that goes beyond my original patch :-/
Would you mind to provide a patch/PR that adds `"can"` before the `channel` only when channel is a simple number?
Thanks!

Reply:
Done

Reply:
Big thanks!